### PR TITLE
Integrate typesense search into dashboard

### PR DESCRIPTION
**Pull request by Sami1309**

### CODE REVIEW

File: dashboard/src/api/resources/Resources.js [hunk 1]
```diff
@@ -109,17 +109,16 @@ export default class ResourcesAPI {
   }
 
   fetchSearch(query) {
-    const fetchAddress = "/data/lists/wine-data.json";
+    const TypesenseClient = require("./Search.js");
+    const myTypesenseClient = new TypesenseClient("8108", "localhost", "xyz");
+    const myResults = myTypesenseClient.search(
+      { q: query, query_by: "Name" },
```
Comment (Sami1309) [inline dashboard/src/api/resources/Resources.js:115]: Add these to class since they are generic

File: dashboard/src/api/resources/Resources.js [hunk 2]
```diff
@@ -109,17 +109,16 @@ export default class ResourcesAPI {
   }
 
   fetchSearch(query) {
-    const fetchAddress = "/data/lists/wine-data.json";
+    const TypesenseClient = require("./Search.js");
+    const myTypesenseClient = new TypesenseClient("8108", "localhost", "xyz");
```
Comment (Sami1309) [inline dashboard/src/api/resources/Resources.js:113]: no need to use "my"

Comment (simba-git) [inline dashboard/src/api/resources/Resources.js:113]: Better var names, also the API key, port, localhost should be somewhere else

Comment (simba-git) [reply dashboard/src/api/resources/Resources.js:113]: Also you should just keep the client created somehwere

File: dashboard/src/api/resources/Resources.js (hunk 1 above)
Comment (simba-git) [inline dashboard/src/api/resources/Resources.js:115]: Hardcode everything except q into search.js

File: dashboard/src/api/resources/Resources.js (hunk 2 above)
Comment (Sami1309) [inline dashboard/src/api/resources/Resources.js:113]: make this a static class inside resources class

**Review by simba-git — CHANGES_REQUESTED** [review]

**Review by simba-git — APPROVED** [review]

**Pull request merged**